src/language-markdown/index.js

```diff
@@ -5,26 +5,37 @@ const options = require("./options");
 const createLanguage = require("../utils/create-language");
 
 const languages = [
-  createLanguage(require("linguist-languages/data/markdown"), {
+  createLanguage(require("linguist-languages/data/Markdown"), {
     override: {
       since: "1.8.0",
       parsers: ["remark"],
-      vscodeLanguageIds: ["markdown"]
+      vscodeLanguageIds: ["markdown"],
+      extensions: [
+        ".md",
+        ".markdown",
+        ".mdown",
+        ".mdwn",
+        ".mkd",
+        ".mkdn",
+        ".mkdown",
+        ".ronn",
+        ".workbook"
```

Comment:
It'd be better to have an `exclude` field for `createLanguage` so that we don't need to manually copy-paste it if there're upstream updates.

Reply:
:+1: